db/migrate/20221211060554_add_address_references.rb

```diff
@@ -0,0 +1,6 @@
+class AddAddressReferences < ActiveRecord::Migration[6.0]
+  def change
+    add_reference :orders, :address, null: true, foreign_key: true
```

Comment:
Have to allow null address references initially, switching to not allow null's, after running the migration to populate addresses for orders and gifts